mart/configs/attack/enforcer/constraints/lp.yaml

```diff
@@ -1,6 +1,6 @@
 lp:
   _target_: mart.attack.enforcer.Lp
-  eps: ???
+  eps: ${....eps}
```

Comment:
How can you make this assumption? You don't really know where this will end up so using `....` is a bad idea.

Reply:
You are right. I should not assume the structure here at the individual component level.

Fixed in [2c5a1b7](https://github.com/IntelLabs/MART/pull/211/commits/2c5a1b7653d4840f8377a498531f5c0bcfe908b8)